Makefile

```diff
@@ -83,10 +84,12 @@ release: all kthxbai docs $(SMF_MANIFESTS)
 		$(TOP)/Makefile \
 		$(TOP)/node_modules \
 		$(TOP)/agent.js \
-		$(TOP)/package.json \
```

Comment:
Dumb question, why remove this?

Reply:
Dumb answer -> because you're generating package.json with the json(1) command that's added.